azure-pipelines/compliance/compliance.yml

```diff
@@ -0,0 +1,49 @@
+steps:
+  - task: ComponentGovernanceComponentDetection@0
+    displayName: 'Component Governance Detection'
+
+  - task: securedevelopmentteam.vss-secure-development-tools.build-task-antimalware.AntiMalware@4
```

Comment:
Also included is an update to basically all of these compliance tasks; there were warnings that the older versions were deprecated.